docs/release_notes.rst

```diff
@@ -10,6 +10,33 @@ This is a major release and contains breaking changes. Please read the notes bel
 * Index count, query, and scan methods are now instance methods.
 
 
+v5.3.0
+----------
+* No longer call ``DescribeTable`` API before first operation
```

Comment:
Wow. This is awesome. Was looking into this sometime back. Did not know this is gone.